core/services/ocr2/plugins/directrequestocr/test/internal/testutils.go

```diff
@@ -0,0 +1,307 @@
+	drconfig "github.com/smartcontractkit/chainlink/core/services/ocr2/plugins/directrequestocr/config"
+	"github.com/smartcontractkit/chainlink/core/services/ocr2/validate"
+	"github.com/smartcontractkit/chainlink/core/services/ocrbootstrap"
+	"github.com/smartcontractkit/chainlink/core/store/models"
+)
+
+func ptr[T any](v T) *T { return &v }
+
+func SetConfig(t *testing.T, owner *bind.TransactOpts, oracleContract *ocr2dr_oracle.OCR2DROracle, oracles []confighelper2.OracleIdentityExtra) {
+	S := make([]int, len(oracles))
+	for i := 0; i < len(S); i++ {
+		S[i] = 1
+	}
+
+	reportingPluginConfigBytes, err := drconfig.EncodeReportingPluginConfig(&drconfig.ReportingPluginConfigWrapper{
+		Config: &drconfig.ReportingPluginConfig{
+			MaxQueryLengthBytes:       1_000,
+			MaxObservationLengthBytes: 1_000,
+			MaxReportLengthBytes:      1_000,
+			MaxRequestBatchSize:       5,
+			DefaultAggregationMethod:  drconfig.AggregationMethod_AGGREGATION_MODE,
+			UniqueReports:             true,
+		},
+	})
+	require.NoError(t, err)
+
+	signersKeys, transmittersAccounts, f, onchainConfig, offchainConfigVersion, offchainConfig, err := confighelper2.ContractSetConfigArgsForTests(
+		2*time.Second,        // deltaProgress
+		1*time.Second,        // deltaResend
+		1*time.Second,        // deltaRound
+		500*time.Millisecond, // deltaGrace
+		2*time.Second,        // deltaStage
+		5,                    // RMax (maxRounds)
+		S,                    // S (schedule of randomized transmission order)
+		oracles,
+		reportingPluginConfigBytes,
+		50*time.Millisecond, // maxDurationQuery
+		50*time.Millisecond, // maxDurationObservation
+		50*time.Millisecond, // maxDurationReport
+		50*time.Millisecond, // maxDurationAccept
+		50*time.Millisecond, // maxDurationTransmit
+		1,                   // f (max faulty oracles)
+		nil,                 // empty onChain config
+	)
+
+	var signers []common.Address
+	var transmitters []common.Address
+	for i := range signersKeys {
+		signers = append(signers, common.BytesToAddress(signersKeys[i]))
+		transmitters = append(transmitters, common.HexToAddress(string(transmittersAccounts[i])))
+	}
+
+	require.NoError(t, err)
+	_, err = oracleContract.SetConfig(
+		owner,
+		signers,
+		transmitters,
+		f,
+		onchainConfig,
+		offchainConfigVersion,
+		offchainConfig,
+	)
+	require.NoError(t, err)
+}
+
+const finalityDepth int = 4
+
+func CommitWithFinality(b *backends.SimulatedBackend) {
+	for i := 0; i < finalityDepth; i++ {
+		b.Commit()
+	}
+}
+
+type deployedClientContract struct {
+	Address  common.Address
+	Contract *ocr2dr_client_example.OCR2DRClientExample
+}
+
+func StartNewChainWithContracts(t *testing.T, nClients int) (*bind.TransactOpts, *backends.SimulatedBackend, *time.Ticker, common.Address, *ocr2dr_oracle.OCR2DROracle, []deployedClientContract) {
+	owner := testutils.MustNewSimTransactor(t)
+	sb := new(big.Int)
+	sb, _ = sb.SetString("100000000000000000000", 10) // 1 eth
```

Comment:
`assets.NewEth(1)`

Reply:
It doesn't work. I need wei but NewEth doesn't convert to wei (it converts to 1).